### PR TITLE
NONAME-153 more modals

### PR DESCRIPTION
**Pull request by K4RP4T**

### CODE REVIEW

File: frontend/src/lib/components/surveys-page/SurveysTable.svelte [hunk 1]
```diff
@@ -66,25 +63,25 @@
 				<td class="info-entry tooltip">
 					{#if entry.uses_cryptographic_module}
 						<i class="material-symbols-rounded">encrypted</i>
-						<span class="tooltip-text {innerWidth <= 1272 ? 'right' : 'left'}"
+						<span class="tooltip-text {innerWidth <= 1239 ? 'right' : 'left'}"
```
Comment (sjerzykiewicz) [inline frontend/src/lib/components/surveys-page/SurveysTable.svelte:66]: we should avoid adding magic numbers, add consts or try to make it width ratio dependant

Comment (yanvoi) [reply frontend/src/lib/components/surveys-page/SurveysTable.svelte:66]: true, named constants are always better

**Comment by sjerzykiewicz** [comment]
I really don’t like the idea of having 5,000 different pixel values for each element. Let’s keep it simple, like in Tailwind CSS. We can just set some constants and use those. Mixing in random values like 608 and 707 just looks messy
image

**Review by sjerzykiewicz — APPROVED** [review]
lgtm!

**Review by mtszlachetka — APPROVED** [review]

**Pull request merged**